requester.py

```diff
@@ -0,0 +1,99 @@
+import requests
+
+# For now request works only for default '24h' forecast period
+
+# weatherapi.com API
+
+
+def request_weatherapi(keys_dict, args):
+    url_wheatherapi = 'https://api.weatherapi.com/v1/forecast.json'
+    API_KEY_wheatherapi = keys_dict['wheatherapi.com']
```

Comment:
URLs and commonly-used variables should be constants.

Reply:
done for all URLs.

Reply:
I see, cool, the only thing is all the constants should be placed at the beginning of the file, before the methods. This way it can be reused by more than one method if needed. Please update that part and it'll be fine.

Reply:
Don't mind the recent comment - there are two kinds of variables you're using, URLs and keys. The first one I believe should be constants and grouped at the beginning of this file, Key values are passed to these methods and are regular parameters passed to the method. I see 3 options for the keys: 
- pass the whole dict and extract only the needed key
- pass the whole dict and pass only the needed key to the submethod
- pass only the needed key to the method already
You can pick whichever way you want.